helm/values.yaml

```diff
@@ -0,0 +1,111 @@
+# ---------------------------------------------------------------------
+# -- Global values
+# ---------------------------------------------------------------------
+image:
+  repository: burningalchemist/sql_exporter
+  pullPolicy:
+    IfNotPresent
+    #   tag: $IMAGE_TAG
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+service:
+  type: ClusterIP
+  port: 80
+resources:
+  {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+podSecurityContext: {}
+securityContext:
+  {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+  # ---------------------------------------------------------------------
+  # -- Prometheus operator valuesnikolai.rodionov@grandcentrix.net
+  # ---------------------------------------------------------------------
```

Comment:
@allanger  This part is a bit confusing to me. But I suggested adding your name in the maintainers list above. 👍

Reply:
Damn, it was an accidental central mouse button click, I guess :)

Reply:
It's gone